public/app/api/utils.ts

```diff
@@ -0,0 +1,3 @@
+import { config } from '@grafana/runtime';
+
+export const getK8sNamespace = () => config.namespace;
```

Comment:
I think this should have a less..... k8s-y name. Feels just a bit too implementation detail-y to me.

maybe `getAPINamespace`?

Reply:
I can't answer for all the other functions in alerting that reference k8s

Reply:
We mostly just settled on the terminology because we call it the `k8s API` in our conversations but no reason to reflect that in our code :D